examples/scroll/README.md

```diff
@@ -0,0 +1,28 @@
+# Scroll Elements Into View
+
+**Test Code**: [index.js](index.js)  
+
+TestCafe [actions](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html) have a built-in scrolling mechanism. In rare cases you might need to scroll the page manually. This example demonstrates two common ways to do this: a `hover` action and a [ClientFunction](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html)
+
+The test page has 2 buttons. They are located 2000 px and 4000 px to the right respectively, both outside of viewport.
+
+The `index.js` file includes 2 tests.
+
+The first test scrolls the page with a [hover action](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/hover.html).
+
+The second test introduces 2 custom Client Functions that scroll the page:
+
+1. The `scrollToElement(selector, offsetX, offsetY)` function scrolls the page so that the element is inside the viewport.
+
+    | Parameter  | Description |
+    |------------|-------------|
+    | `selector` | a selector that identifies page element to scroll to |
```

Comment:
you can't identify something to do something

are you trying to say - a selector that identifies the page element used to scroll

I don't really know what you want to say

Reply:
We identify an element. Then TestCafe scrolls the page until it reaches said element. 
Is that better:

> a selector that identifies page element TestCafe should scroll to

Reply:
> a selector that identifies the page element